src/Glpi/Controller/InventoryController.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Controller;
+
+use Glpi\Exception\Http\AccessDeniedHttpException;
+use Glpi\Http\Firewall;
+use Symfony\Component\HttpFoundation\Request;
+use Glpi\Inventory\Conf;
+use Glpi\Security\Attribute\SecurityStrategy;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+final class InventoryController extends AbstractController
+{
+    public static bool $is_inventory = false;
+    #[SecurityStrategy(Firewall::STRATEGY_NO_CHECK)]
+    #[Route("/Inventory", name: "glpi_inventory", methods: ['GET', 'POST'])]
+    #[Route("/front/inventory.php", name: "glpi_inventory_legacy", methods: ['GET', 'POST'])]
+    public function __invoke(Request $request): Response
+    {
+        $conf = new Conf();
+        self::$is_inventory = true;
```

Comment:
The entire function should be under a `try/catch` block, so that assigning this static variable to `false` can be done in a `finally` block to ensure it's properly deactivated at the right moment.

Reply:
Indeed it may be better, I'll change

Reply:
No sure finally, since it may require extra changes out of the scope of this PR. I guess static variable will be reset to false on each controller cal, no? 
I do not see any real case where it really can cause an issue.

Reply:
> I guess static variable will be reset to false on each controller cal, no?

There is no place where the variable is set to false, and since it's static, it's global and therefore set at autoload time, so yes, this try/finally is unfortunately mandatory or the variable will never be set to false in the same process, making it completely untestable without e2e testing, and error-prone for potential other usage (like listeners for example)